python/practices/practice-3/src/path1/word_e_checker.py

```diff
@@ -0,0 +1,35 @@
+"""
+Count and print percentage word in word list not contains the letter 'e' or 'E'
+"""
+
+
+import os
+
+
+def count_words():
+    """
+    Count and calculate the percentage of words that do not contain 'E' or 'e'
+    in a list of words.
+    """
+
+    words = input("Enter the list of words: ").split()
+    count_word = len(words)
+    words_has_no_e = 0
+
+    for word in words:
+        if 'e' not in word and 'E' not in word:
+            print(word)
+            words_has_no_e += 1
+
+    percentage = (words_has_no_e/count_word) * 100
```

Comment:
The function name is count_wort, it should only do the counting, statistical work should be done in another function.

Reply:
Done